src/electos/ballotmaker/constants.py

```diff
@@ -0,0 +1,9 @@
+"""Avoid using magic numbers with these handy constants
+   Use the same values in tests, function returns. 
+"""
+
+PROGRAM_NAME = "BallotMaker"
+VERSION = "0.1.1"
+
+NO_ERRORS = 0
+NO_FILE = -10
```

Comment:
Since `main` is returning these values to the calling program use [`errno`](https://docs.python.org/3.8/library/errno.html) and don't define your own. For example returning 10 means `ECHILD` (no child processes) which is not what you intend.

```suggestion
NO_FILE = errno.ENOENT
```

Reply:
The `errno` module is exactly what I need here. Today I learned...